plugins/analytics/public/threatminer.py

```diff
@@ -1,12 +1,36 @@
 import logging
 import requests
 import json
+from core.errors import GenericYetiError
 from core.analytics import OneShotAnalytics
 from core.observables import Hash, Ip, Hostname, Url
 from core.config.config import yeti_config
 from core.errors import ObservableValidationError
 from hashlib import sha1
 
+def aux_checker(json_result):
+    if not json_result:
```

Comment:
You can also raise a GenericYetiError here, which takes away lots of confusing handling of empty return variables.

```python
if not json_result or not json_result.get("results"): # this way you also take care of the check in L30
  raise GenericYetiError("blah")

json_string = json.dumps(...)
# ...
return _results, result
```